gnocchi/cli/manage.py

```diff
@@ -105,3 +105,7 @@ def change_sack_size():
     s.remove_sacks()
     LOG.info("Creating new %d sacks", conf.sacks_number)
     s.upgrade(conf.sacks_number)
+
+
+if __name__ == '__main__':
+    upgrade()
```

Comment:
i don't really have any concerns with proposal. to be nitpicky, it is a bit opaque that manage.py would upgrade. also, the commands in this file are ones you wouldn't want to trigger by accidental. if not critical to your goal, i would probably skip at least this file.

Reply:
I see. What if we create an "upgrade.py", that has the main and calls the upgrade from manage.py? Then, we avoid this kind of issue, and make the module explicit.

Reply:
this is just me being paranoid but i've destroyed/corrupted the wrong db before because i wasn't thinking or i had command in my terminal history.

this is to minimise chance of someone triggering upgrade by accident. granted you still need to have a conf file and i don't know think this is that much more unsafe than having gnocchi-upgrade cmd installed. so not 100% against it but with my ops hat on, this is not idiot-proof. :)

 i don't have better suggestion if i'm honest so i will approve.

Reply:
ok. Thanks @chungg for the review!